dev/com.ibm.ws.security.kerberos.auth/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,29 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#CMVCPATHNAME com.ibm.ws.security.kerberos.auth/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+kerberos=Kerberos
+kerberos.desc=System-wide settings for Kerberos authentication
+
+keytab=Kerberos keytab file
+keytab.desc=The path of the keytab file that is used to obtain the principal's secret key. If unspecified, the keytab from \
+the Kerberos configuration file is used. If a keytab file is not specified in the Kerberos configuration file, then the location \
+{user.home}/krb5.keytab is checked.
+
+configFile=Kerberos conf file
+configFile.desc=The location of an MIT style krb5.conf configuration file. The location is passed to the JDK implementation of \
+the Kerberos APIs using the java.security.krb5.conf system property. If unspecified, the krb5.conf file is searched for according to the \
```

Comment:
**change**
The location is passed to the JDK implementation of \
the Kerberos APIs using the java.security.krb5.conf system property.

depending on which of the following is correct:
**to**
The location is passed to the JDK implementation of \
the Kerberos APIs by using the java.security.krb5.conf system property.
**or**
The location is passed to the JDK implementation of \
the Kerberos APIs that are using the java.security.krb5.conf system property.